src/frontend/planner_test/tests/testdata/range_scan.yaml

```diff
@@ -233,28 +233,94 @@
   batch_plan: |
     BatchExchange { order: [], dist: Single }
     └─BatchScan { table: orders_count_by_user_ordered, columns: [orders_count_by_user_ordered.user_id, orders_count_by_user_ordered.date, orders_count_by_user_ordered.orders_count], scan_ranges: [orders_count_by_user_ordered.orders_count = Int64(10)], distribution: UpstreamHashShard(orders_count_by_user_ordered.user_id, orders_count_by_user_ordered.date) }
+- name: Cannot push down for Mutiple lower bound 1.
+  before:
+  - create_table_and_mv_ordered
+  sql: |
+    SELECT * FROM orders_count_by_user_ordered WHERE user_id < 10 and user_id < 30
```

Comment:
```suggestion
- name: Cannot push down for Mutiple upper bound 1.
  before:
  - create_table_and_mv_ordered
  sql: |
    SELECT * FROM orders_count_by_user_ordered WHERE user_id < 10 and user_id < 30
```
(same below)

Reply:
Sorry, my fault...I misuse lower bound and upper bound.